ladybug/src/main/resources/springIbisDebuggerAdvice.xml

```diff
@@ -149,7 +168,7 @@
 					args(message, session, next)
 					"
 				method="debugStreamingSenderInputOutputAbort"
-				arg-names="blockHandle, message, session, next"
+				arg-names="message, session, next"
```

Comment:
goed dat je dit gevonden hebt.
We moeten echt af van dat ge-AOP waar het ook programmatisch kan!